### ISSUE TITLE
Invalid Link

### ISSUE THREAD

**Issue by SpEcHiDe** [fossasia/loklak_search#128]
A link in this page: http://loklak.fossasia.org/contact points to an invalid location!
```
Please use our [contact](http://loklak.fossasia.org/) form to report the issue.
```
It should have pointed to a contact form, if it exists!

**Comment by hemantjadon** [comment]
@SpEcHiDe 
Thanks for ponting this out Cheers :)

@mariobehling Can you please point out where it should point actually? (Parent issue #61 ).

**Comment by hemantjadon** [comment]
@mariobehling Can you point out where should this link point.
![loklak search](https://cloud.githubusercontent.com/assets/11755543/21466799/c74d54f2-c9fb-11e6-83d5-a317859b05c8.png)

**Comment by mariobehling** [comment]
@hemantjadon We should have a contact form that opens in a lightbox here. I opened an issue: https://github.com/fossasia/loklak_search/issues/152

**Comment by hemantjadon** [comment]
Okay cool :) So should this be closed??

**Comment by hemantjadon** [comment]
This is fixed now in PR #162
